src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj

```diff
@@ -222,7 +223,8 @@
                            ssl-config)
                         :client-auth :none
                         :cipher-suites (or (:cipher-suites ssl-config) config/acceptable-ciphers)
-                        :protocols     (or (:protocols ssl-config) config/default-protocols)}))
+                        :protocols     (or (:protocols ssl-config) config/default-protocols)
+                        :cache-session-id (or (:cache-session-id ssl-config) true)}))
```

Comment:
Ah, you're right. I'm also wondering if we should add a proxy test exercising this piece of functionality.